core/src/main/scala/org/apache/spark/rdd/PairRDDFunctions.scala

```diff
@@ -1053,7 +1053,10 @@ class PairRDDFunctions[K, V](self: RDD[(K, V)])
     // users that they may loss data if they are using a direct output committer.
     val speculationEnabled = self.conf.getBoolean("spark.speculation", false)
     val outputCommitterClass = hadoopConf.get("mapred.output.committer.class", "")
-    if (speculationEnabled && outputCommitterClass.contains("Direct")) {
+    val outputCommitCoordinationEnabled = self.conf.getBoolean(
+      "spark.hadoop.outputCommitCoordination.enabled", true)
```

Comment:
since we are touching it, can we define this config in `org.apache.spark.internal.config`?

Reply:
Update @cloud-fan Thanks